helm-chart/kuberay-operator/templates/deployment.yaml

```diff
@@ -61,6 +69,18 @@ spec:
             {{- $argList = append $argList "--watch-namespace" -}}
             {{- $argList = append $argList $watchNamespace -}}
             {{- end -}}
+            {{- if and (.Values.logging.baseDir) (.Values.logging.fileName) -}}
+            {{- $argList = append $argList "--log-file-path" -}}
+            {{- $argList = append $argList (printf "%s/%s" .Values.logging.baseDir .Values.logging.fileName) -}}
+            {{- end -}}
+            {{- if .Values.logging.stdoutEncoder -}}
+            {{- $argList = append $argList "--zap-encoder" -}}
```

Comment:
Do we have this flag?

Reply:
Yes, this is inherited from controller-runtime https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/log/zap/zap.go#L266

Reply:
Do we need to add something like [1] in `main.go` for `zap-encoder`?

```go
	flag.StringVar(&logFileEncoder, "log-file-encoder", "json",
		"Encoder to use for log file. Valid values are 'json' and 'console'. Defaults to 'json'")
```

Reply:
@kevin85421 the definition for `zap-encoder` is in controller-runtime already. Unless you mean to redefine the same flag in main.go (e.g. `--stdout-encoder`) that overrides --zap-encoder

Reply:
I think I have a gap in understanding because I didn't grasp how the flag is passed to the logger.

* The flag seems to be passed by `flag.CommandLine`.
https://github.com/ray-project/kuberay/blob/2c26cff24c8bb24cfbe4391eeeea30a1135f6bf9/ray-operator/main.go#L94-L95
* https://sdk.operatorframework.io/docs/building-operators/golang/references/logging/